internal/report/report_test.go

```diff
@@ -210,14 +212,81 @@ func TestWriter_calculateExitCode(t *testing.T) {
 			},
 			want: ExitCodeCheckError,
 		},
+		{
+			name: "stale exclusions (warn)",
+			summ: summary{
+				count: map[config.Severity]int{
+					config.SeverityCritical: 0,
+					config.SeverityHigh:     0,
+					config.SeverityMedium:   1,
+					config.SeverityLow:      1,
+					config.SeverityInfo:     1,
+				},
+			},
+			status: []checkStatus{
+				{
+					Checktype: "Checktype1",
+					Target:    "Target1",
+					Status:    "FINISHED",
+				},
+			},
+			staleExcls: []config.Exclusion{
+				{
+					Summary: "Unused exclusion",
+				},
+			},
+			rConfig: config.ReportConfig{
+				Severity: config.SeverityHigh,
+				Exclusions: []config.Exclusion{
+					{
+						Summary: "Unused exclusion",
+					},
+				},
+			},
+			want: 0,
+		},
+		{
+			name: "stale exclusions (error)",
+			summ: summary{
+				count: map[config.Severity]int{
+					config.SeverityCritical: 0,
+					config.SeverityHigh:     0,
+					config.SeverityMedium:   1,
+					config.SeverityLow:      1,
+					config.SeverityInfo:     1,
+				},
+			},
+			status: []checkStatus{
+				{
+					Checktype: "Checktype1",
+					Target:    "Target1",
+					Status:    "FINISHED",
+				},
+			},
+			staleExcls: []config.Exclusion{
+				{
+					Summary: "Unused exclusion",
+				},
+			},
+			rConfig: config.ReportConfig{
+				Severity:               config.SeverityHigh,
+				ErrorOnStaleExclusions: true,
+				Exclusions: []config.Exclusion{
+					{
+						Summary: "Unused exclusion",
+					},
+				},
+			},
+			want: 4,
```

Comment:
I'd use the defined constant.

```suggestion
			want: ExitCodeStaleExclusions,
```